assets/eip-6353/contracts/ERC20Charity.sol

```diff
@@ -0,0 +1,328 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
I would very strongly recommend going through and implementing this _without_ OpenZeppelin. It will give a more concise Reference Implementation, and will give you better insight into what you need to put in your standard. Right now I think you depend too much on the OZ framework for someone to come along and implement this EIP from scratch.

Reply:
I'll try to do that.